clients/admin-ui/src/features/system/system.slice.ts

```diff
@@ -120,10 +121,17 @@ export const systemSlice = createSlice({
     ) => {
       draftState.activeSystem = action.payload;
     },
+    setActiveClassifySystemFidesKey: (
+      draftState,
+      action: PayloadAction<string | undefined>
+    ) => {
+      draftState.activeClassifySystemFidesKey = action.payload;
```

Comment:
Is this fides key inherently the system for classify purposes? With dataset+classify there's a single `activeDatasetFidesKey` so everything is in sync. Though I suppose if non-classify code still wants the _whole_ system object that would require a bigger refactor.

Reply:
If I'm interpreting your question right, you're asking if there's a difference between `activeSystem` and `activeClassifySystem`? there is right now since `activeSystem` is for editing a system from the `/system` page, while `activeClassifySystem` is for editing a system from the `/classify-systems` page. Those two pages aren't related right now (`/system` is just the CRUD for systems, while `/classify-systems` comes after a scan), but perhaps should be, with more design/product thought